apps/crn-frontend/src/network/teams/__tests__/TeamProfile.test.tsx

```diff
@@ -815,3 +815,68 @@ describe('The draft output tab', () => {
     expect(screen.getByText('Draft Outputs (0)')).toBeVisible();
   });
 });
+
+describe('The compliance tab', () => {
+  it('does not show compliance tab if not on Team ASAP', async () => {
+    enable('DISPLAY_MANUSCRIPTS');
+    await renderPage({
+      ...createTeamResponse(),
+      displayName: 'Test',
+    });
+
+    expect(
+      screen.queryByText(/Compliance/i, { selector: 'nav *' }),
+    ).not.toBeInTheDocument();
+  });
+
+  it('does not show compliance tab if on Team ASAP but not Staff', async () => {
+    enable('DISPLAY_MANUSCRIPTS');
+    await renderPage(
+      {
+        ...createTeamResponse(),
+        displayName: 'Test',
```

Comment:
```suggestion
        displayName: 'ASAP',
```

I'm suggesting this change because the title of the test says "if on Team ASAP"